### PR TITLE
Add arachni image

### PR DESCRIPTION
**Pull request by tjomk**

### CODE REVIEW

File: arachni/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM unit9/base:latest
+
+MAINTAINER Artiom Vasiliev <artiom@unit9.com>
+
+RUN cd /tmp && \
```
Comment (rollcat) [inline arachni/Dockerfile]: `&&` goes at the beginning of the next line

File: arachni/Makefile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+NAME := arachni
+VERSION: = 1.0.0
```
Comment (rollcat) [inline arachni/Makefile]: 1. typo (`: =` vs ` :=`)
2. Not sure if hardcoding `VERSION` in Makefile makes sense? Dockerhub will normally build and tag `latest` (per [howto](https://github.com/unit9/docklabs/blob/master/howto.md)); also we don't really have a policy for versioning images.

**Review by rollcat — APPROVED** [review]
:+1: sans minor style issues

File: arachni/Dockerfile (hunk 1 above)
Comment (tjomk) [reply arachni/Dockerfile]: old habits are not easy to get rid of...

File: arachni/Makefile (hunk 2 above)
Comment (tjomk) [reply arachni/Makefile]: I caught it during testing but haven't submitted.

Maybe worth discussing versioning some time? k8s doesn't like to constantly deploy `latest` but I'm not sure how that ties to auto builds on docker hub.

**Pull request merged**